pkgs/servers/blockbook/default.nix

```diff
@@ -14,14 +14,14 @@
 
 buildGoModule rec {
   pname = "blockbook";
-  version = "0.3.3";
+  version = "0.3.4";
   commit = "b6961ca";
```

Comment:
```suggestion
  commit = "eb4e10a";
```

Reply:
I know this breaks automatic updates, but these should not occur often (1-2 times per year). If it's too much of a hassle, we can drop the `commit` field.

It's used by the application to get a clickable link that leads to upstream:
![image](https://user-images.githubusercontent.com/791309/84647185-61469600-af03-11ea-90ca-ead26d885471.png)

So it's nice, but definitely not essential.